### ISSUE TITLE
Sleeping Dogs (GoG) requires directory prior to install or it will fail

### ISSUE THREAD

**Issue by AndyTheNerd** [Heroic-Games-Launcher/HeroicGamesLauncher#3632]
### Describe the bug

Sleeping Dogs (GoG version) requires a pre-existing directory of `Sleeping Dogs Definitive Edition` prior to installing it via Heroic. If you do not manually set up this directory, the install will hang and eventually fail. The game will then show as "canceled" in Completed list of games in Download Manager.

Logs will show an error of
```
Installation of 1741843789 failed with: Error: ENOENT: no such file or directory, stat 'D:\Heroic Games\Sleeping Dogs Definitive Edition'
``` 

If the directory is manually created by the user, and the user selects the directory above this prior to install, the game will install as expected. So for example, a user will need to create a directory `D:\Heroic Games\Sleeping Dogs Definitive Edition` and then select the directory `D:\Heroic Games\` when installing the game.

This issue was reproduced both on Windows 11 and Steam Deck. Workaround fixes the issue on both environments.

### Add logs

```text
(13:27:38) INFO:    [Legendary]:        Legendary location: C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary
(13:27:38) INFO:    [Gog]:              GOGDL location: C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl
(13:27:38) INFO:    [Connection]:       Connectivity: check-online
(13:27:38) INFO:    [Connection]:       Pinging external endpoints
(13:27:38) INFO:    [Gog]:              Checking for existing gog manifests
(13:27:38) INFO:    [Nile]:             Refreshing library...
(13:27:38) INFO:    [Nile]:             Refreshing Amazon Games...
(13:27:38) INFO:    [Nile]:             Game list updated, got 138 games
(13:27:38) INFO:    [Backend]:          DRM module staus {
  "neifaoindggfcjicffkgpmnlppeffabd": {
    "name": "Google Widevine Windows CDM",
    "status": "new",
    "version": "1.0.2738.0"
  },
  "oimompecagnajdejgnnjijobebaeigek": {
    "name": "Widevine Content Decryption Module",
    "status": "new",
    "version": "4.10.2710.0"
  }
}
(13:27:39) WARNING: [Backend]:          Protocol already registered.
(13:27:39) INFO:    [Nile]:             Running command: NILE_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\nile_config powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\nile`"" -Wait -ArgumentList "`"library`"","`"sync`"" -NoNewWindow
(13:27:39) INFO:    [Connection]:       Connectivity: online
(13:27:39) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(13:27:39) INFO:    [Gog]:              Checking if login is valid
(13:27:39) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(13:27:39) INFO:    [Gog]:              Updating redist, reason - different number of redist
(13:27:39) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(13:27:39) INFO:    [Backend]:          VCRuntime is installed
(13:27:39) INFO:    [Legendary]:        Running command: LEGENDARY_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary`"" -Wait -ArgumentList "`"--version`"" -NoNewWindow
(13:27:39) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"--version`"" -NoNewWindow
(13:27:39) INFO:    [Nile]:             Running command: NILE_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\nile_config powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\nile`"" -Wait -ArgumentList "`"--version`"" -NoNewWindow
(13:27:39) INFO:    [Nile]:             Saved user data to config file
(13:27:39) INFO:    [Frontend]:         Refreshing undefined Library
(13:27:39) WARNING: [Backend]:          refresh not implemented on Sideload Library Manager
(13:27:39) INFO:    [Legendary]:        Refreshing library...
(13:27:39) INFO:    [Legendary]:        Refreshing Epic Games...
(13:27:39) INFO:    [Nile]:             Refreshing library...
(13:27:39) INFO:    [Nile]:             Refreshing Amazon Games...
(13:27:39) INFO:    [Nile]:             Running command: NILE_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\nile_config powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\nile`"" -Wait -ArgumentList "`"library`"","`"sync`"" -NoNewWindow
(13:27:39) INFO:    [Nile]:             Game list updated, got 138 games
(13:27:39) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(13:27:39) INFO:    [Backend]:          Frontend Ready
(13:27:39) INFO:    [Backend]:          Checking for new Heroic Updates
(13:27:40) INFO:    [Backend]:          Checking for current version changelog
(13:27:40) INFO:    [Legendary]:        Running command: LEGENDARY_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary`"" -Wait -ArgumentList "`"list`"","`"--third-party`"" -NoNewWindow
(13:27:42) INFO:    [WineDownloader]:   Updating wine versions info
(13:27:42) INFO:    [WineDownloader]:   Fetching upstream information...
(13:27:42) INFO:    [Gog]:              Getting game metadata: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"info`"","`"gog-redist`"","`"--os`"","`"windows`"" -NoNewWindow
(13:27:42) INFO:    [Gog]:              Getting GOG library
(13:27:42) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(13:27:42) INFO:    [Backend]:          
System Information:
CPU: 16x 11th Gen Intel(R) Core(TM) i7-11700K @ 3.60GHz
Memory: 34.25 GB (used: 8.94 GB)
GPUs:
  GPU 0:
    Name: NVIDIA Corporation GA102 [GeForce RTX 3080 Lite Hash Rate]
    IDs: D=2216 V=10de SD=403f SV=1458
    Driver: 551.76
OS: Microsoft Windows 11 Home 10.0.22621 (win32)

The current system is not a Steam Deck
We are not running inside a Flatpak container

Software Versions:
  Heroic: 2.13.0 Dorry & Broggy
  Legendary: 0.20.34 Direct Intervention
  gogdl: 1.0.0
  Nile: 1.0.1 Jonathan Joestar

(13:27:42) INFO:    [Gog]:              Saved username to config file
(13:27:43) INFO:    [WineDownloader]:   wine versions updated
(13:27:43) ERROR:   [Backend]:          Could not find default wine version
(13:27:43) ERROR:   [Gog]:              stdout = false and res.abort = false in getInstallInfo
(13:27:43) INFO:    [DownloadManager]:  Galaxy Common Redistributables  was added to the download queue.
(13:27:43) INFO:    [Gog]:              Updating GOG redistributables
(13:27:43) INFO:    [Gog]:              GOG REDIST:: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"redist`"","`"--ids`"","`"ISI`"","`"--path`"","`"C:\Users\andre\AppData\Roaming\heroic\tools\redist\gog`"" -NoNewWindow
(13:27:43) DEBUG:   [Gog]:              Logging to file "C:\Users\andre\AppData\Roaming\heroic\GamesConfig\gog-redist.log"
(13:27:43) INFO:    [Legendary]:        Game list updated, got 277 games & DLCs
(13:27:43) INFO:    [Backend]:          Preventing machine to sleep
(13:27:44) INFO:    [Backend]:          Starting the Download Queue
(13:27:44) INFO:    [Gog]:              Updating GOG redistributables
(13:27:44) INFO:    [Gog]:              GOG REDIST:: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"redist`"","`"--ids`"","`"ISI`"","`"--path`"","`"C:\Users\andre\AppData\Roaming\heroic\tools\redist\gog`"" -NoNewWindow
(13:27:44) DEBUG:   [Gog]:              Logging to file "C:\Users\andre\AppData\Roaming\heroic\GamesConfig\gog-redist.log"
(13:27:45) INFO:    [Gog]:              Saved games data
(13:27:45) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(13:27:45) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(13:27:46) INFO:    [DownloadManager]:  gog-redist added to download manager finished.
(13:27:46) INFO:    [DownloadManager]:  gog-redist removed from download manager.
(13:27:46) INFO:    [DownloadManager]:  gog-redist added to download manager finished.
(13:27:46) INFO:    [DownloadManager]:  gog-redist removed from download manager.
(13:27:46) INFO:    [Backend]:          operation interrupted
(13:27:46) INFO:    [ExtraGameInfo]:    Using cached ExtraGameInfo data for Sleeping Dogs Definitive Edition
(13:27:46) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(13:27:46) INFO:    [ExtraGameInfo]:    Using cached ExtraGameInfo data for Sleeping Dogs Definitive Edition
(13:27:46) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(13:27:46) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(13:27:47) INFO:    [Gog]:              Found 0 game(s) to update
(13:27:47) INFO:    [Legendary]:        Checking for game updates: LEGENDARY_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary`"" -Wait -ArgumentList "`"list`"","`"--third-party`"" -NoNewWindow
(13:27:48) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(13:27:49) DEBUG:   [Backend]:          {"free":603644522496,"diskSize":2000381014016,"message":"562.19 GiB / 1.82 TiB","validPath":true,"validFlatpakPath":true}
(13:27:49) DEBUG:   [Backend]:          {"free":603644522496,"diskSize":2000381014016,"message":"562.19 GiB / 1.82 TiB","validPath":true,"validFlatpakPath":true}
(13:27:49) INFO:    [Legendary]:        Found 0 games to update
(13:27:49) INFO:    [Nile]:             Looking for updates...
(13:27:49) INFO:    [Nile]:             Running command: NILE_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\nile_config powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\nile`"" -Wait -ArgumentList "`"list-updates`"","`"--json`"" -NoNewWindow
(13:27:50) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(13:27:50) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(13:27:53) DEBUG:   [Backend]:          {"free":603644522496,"diskSize":2000381014016,"message":"562.19 GiB / 1.82 TiB","validPath":true,"validFlatpakPath":true}
(13:27:54) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(13:27:54) INFO:    [DownloadManager]:  Sleeping Dogs: Definitive Edition  was added to the download queue.
(13:27:54) INFO:    [Backend]:          Preventing machine to sleep
(13:27:54) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(13:27:54) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(13:27:54) INFO:    [Gog]:              Updating GOG redistributables
(13:27:54) INFO:    [Gog]:              GOG REDIST:: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"redist`"","`"--ids`"","`"ISI,DirectX,MSVC2010,MSVC2019`"","`"--path`"","`"C:\Users\andre\AppData\Roaming\heroic\tools\redist\gog`"" -NoNewWindow
(13:27:54) DEBUG:   [Gog]:              Logging to file "C:\Users\andre\AppData\Roaming\heroic\GamesConfig\gog-redist.log"
(13:27:55) INFO:    [Gog]:              Progress for Galaxy Common Redistributables: 0%/0.00MB/00:00:00 Down: 0MB/s / Disk: 0MB/s
(13:27:56) INFO:    [Gog]:              Progress for Galaxy Common Redistributables: 13.55%/52.21MB/00:00:06 Down: 56.17MB/s / Disk: 16.23MB/s
(13:27:57) INFO:    [Gog]:              Progress for Galaxy Common Redistributables: 81.92%/95.98MB/00:00:00 Down: 56.25MB/s / Disk: 81.9MB/s
(13:27:58) INFO:    [Gog]:              Progress for Galaxy Common Redistributables: 100%/117.38MB/00:00:00 Down: 4.9MB/s / Disk: 21.66MB/s
(13:27:59) INFO:    [DownloadManager]:  gog-redist added to download manager finished.
(13:27:59) INFO:    [DownloadManager]:  gog-redist removed from download manager.
(13:27:59) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(13:27:59) INFO:    [Frontend]:         Refreshing gog Library
(13:27:59) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(13:27:59) INFO:    [Backend]:          operation interrupted
(13:28:00) INFO:    [Gog]:              Installing 1741843789: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"download`"","`"1741843789`"","`"--platform`"","`"windows`"","`"--path`"","`"D:\Heroic Games`"","`"--support`"","`"C:\Users\andre\AppData\Roaming\heroic\gogdlConfig\heroic_gogdl\gog-support\1741843789`"","`"--skip-dlcs`"","`"--lang`"","`"en-US`"" -NoNewWindow
(13:28:00) DEBUG:   [Gog]:              Logging to file "C:\Users\andre\AppData\Roaming\heroic\GamesConfig\1741843789.log"
(13:28:00) INFO:    [Gog]:              Getting GOG library
(13:28:00) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(13:28:03) INFO:    [Gog]:              Saved games data
(13:28:03) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(13:28:03) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(13:28:04) INFO:    [Gog]:              Found 0 game(s) to update
(13:28:04) INFO:    [Legendary]:        Checking for game updates: LEGENDARY_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary`"" -Wait -ArgumentList "`"list`"","`"--third-party`"" -NoNewWindow
(13:28:07) INFO:    [Legendary]:        Found 0 games to update
(13:28:07) INFO:    [Nile]:             Looking for updates...
(13:28:07) INFO:    [Nile]:             Running command: NILE_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\nile_config powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\nile`"" -Wait -ArgumentList "`"list-updates`"","`"--json`"" -NoNewWindow
(13:35:30) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(13:35:30) ERROR:   [DownloadManager]:  Installation of 1741843789 failed with: Error: ENOENT: no such file or directory, stat 'D:\Heroic Games\Sleeping Dogs Definitive Edition'
(13:35:30) WARNING: [DownloadManager]:  Installation of 1741843789 failed!
(13:35:30) INFO:    [DownloadManager]:  1741843789 added to download manager finished.
(13:35:30) INFO:    [DownloadManager]:  1741843789 removed from download manager.
(13:35:30) INFO:    [Frontend]:         Refreshing gog Library
(13:35:30) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(13:35:30) INFO:    [Backend]:          Stopping Power Saver Blocker
(13:35:30) INFO:    [Backend]:          operation interrupted
(13:35:31) INFO:    [Gog]:              Getting GOG library
(13:35:31) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(13:35:34) INFO:    [Gog]:              Saved games data
(16:10:46) INFO:    [ExtraGameInfo]:    Using cached ExtraGameInfo data for Sleeping Dogs Definitive Edition
(16:10:46) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(16:10:46) INFO:    [ExtraGameInfo]:    Using cached ExtraGameInfo data for Sleeping Dogs Definitive Edition
(16:10:46) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(16:10:46) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(16:10:47) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(16:10:48) DEBUG:   [Backend]:          {"free":603641741312,"diskSize":2000381014016,"message":"562.19 GiB / 1.82 TiB","validPath":true,"validFlatpakPath":true}
(16:10:48) DEBUG:   [Backend]:          {"free":603641741312,"diskSize":2000381014016,"message":"562.19 GiB / 1.82 TiB","validPath":true,"validFlatpakPath":true}
(16:10:56) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(16:10:56) INFO:    [DownloadManager]:  Sleeping Dogs: Definitive Edition  was added to the download queue.
(16:10:56) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(16:10:56) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(16:10:56) INFO:    [Gog]:              Updating GOG redistributables
(16:10:56) INFO:    [Gog]:              GOG REDIST:: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"redist`"","`"--ids`"","`"ISI,DirectX,MSVC2010,MSVC2019`"","`"--path`"","`"C:\Users\andre\AppData\Roaming\heroic\tools\redist\gog`"" -NoNewWindow
(16:10:56) DEBUG:   [Gog]:              Logging to file "C:\Users\andre\AppData\Roaming\heroic\GamesConfig\gog-redist.log"
(16:10:58) INFO:    [DownloadManager]:  gog-redist added to download manager finished.
(16:10:58) INFO:    [DownloadManager]:  gog-redist removed from download manager.
(16:10:58) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(16:10:58) INFO:    [Frontend]:         Refreshing gog Library
(16:10:58) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(16:10:58) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(16:10:58) INFO:    [Backend]:          operation interrupted
(16:10:58) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(16:10:59) INFO:    [Gog]:              Installing 1741843789: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"download`"","`"1741843789`"","`"--platform`"","`"windows`"","`"--path`"","`"D:\`"","`"--support`"","`"C:\Users\andre\AppData\Roaming\heroic\gogdlConfig\heroic_gogdl\gog-support\1741843789`"","`"--skip-dlcs`"","`"--lang`"","`"en-US`"" -NoNewWindow
(16:10:59) DEBUG:   [Gog]:              Logging to file "C:\Users\andre\AppData\Roaming\heroic\GamesConfig\1741843789.log"
(16:10:59) INFO:    [Gog]:              Getting GOG library
(16:10:59) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(16:11:01) INFO:    [Gog]:              Got install info from cache for 1741843789 on windows platform
(16:11:01) ERROR:   [DownloadManager]:  Installation of 1741843789 failed with: Error: ENOENT: no such file or directory, stat 'D:\Sleeping Dogs Definitive Edition'
(16:11:01) WARNING: [DownloadManager]:  Installation of 1741843789 failed!
(16:11:01) INFO:    [DownloadManager]:  1741843789 added to download manager finished.
(16:11:01) INFO:    [DownloadManager]:  1741843789 removed from download manager.
(16:11:01) INFO:    [Backend]:          Stopping Power Saver Blocker
(16:11:01) INFO:    [Backend]:          operation interrupted
(16:11:02) INFO:    [Gog]:              Saved games data
(16:11:02) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(16:11:02) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -ArgumentList "`"--auth-config-path`"","`"C:\Users\andre\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`"" -NoNewWindow
(16:11:04) INFO:    [Gog]:              Found 0 game(s) to update
(16:11:04) INFO:    [Legendary]:        Checking for game updates: LEGENDARY_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary`"" -Wait -ArgumentList "`"list`"","`"--third-party`"" -NoNewWindow
(16:11:04) INFO:    [Legendary]:        installed.json updated, refreshing library
(16:11:07) INFO:    [Legendary]:        Found 0 games to update
(16:11:07) INFO:    [Nile]:             Looking for updates...
(16:11:07) INFO:    [Nile]:             Running command: NILE_CONFIG_PATH=C:\Users\andre\AppData\Roaming\heroic\nile_config powershell Start-Process "`"C:\Users\andre\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\nile`"" -Wait -ArgumentList "`"list-updates`"","`"--json`"" -NoNewWindow
```

### Steps to reproduce

1. Do **not** create `Sleeping Dogs` directory prior to install
2. Go to Heroic Launcher
3. Click on Sleeping Dogs (GoG Version) and install it
4. Wait about 30 seconds
5. See error in logs

### Expected behavior

1. Go to Heroic Launcher
2. Click on Sleeping Dogs (GoG Version) and install it
3. Game installs as expected

### Screenshots

![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/30298320/758b4f29-c690-408e-b3d2-806b6acb75a3)

### Heroic Version

Latest Stable

### System Information

- OS : Windows 11 and Steam Deck

### Additional information

This seemingly is only impacting Sleeping Dogs from my testing. Other games tested from my library are installing as expected.

**Comment by imLinguin** [comment]
I think the issue here is in how we spawn the processes on Windows.
Could you try installing the game in some subdirectory? Like `D:\GOG Games`

**Comment by AndyTheNerd** [comment]
@imLinguin You are correct. Created a new subdirectory `D:\Heroic Games` and attempted install there and was successful. Seems to fail unless a subdirectory exists then?

**Comment by imLinguin** [comment]
The issue is the trailing `\` as then powershell treats it as escaping the quote after it

**Comment by AndyTheNerd** [comment]
Would that confirm this bug, or would this be an unforeseen error from a user who does not install games in a subdirectory?

**Comment by arielj** [comment]
I understand this bug was fixed in 2.15.0

please feel free to comment if it wasn't and we can reopen the issue
